packages/runtime/datastore/src/channelStorageService.ts

```diff
@@ -17,14 +17,12 @@ export class ChannelStorageService implements IChannelStorageService {
 		tree: ISnapshotTree,
 		results: { [path: string]: string },
 	) {
-		// eslint-disable-next-line guard-for-in, no-restricted-syntax
-		for (const path in tree.trees) {
-			ChannelStorageService.flattenTree(`${base}${path}/`, tree.trees[path], results);
+		for (const [path, subtree] of Object.entries(tree.trees)) {
+			ChannelStorageService.flattenTree(`${base}${path}/`, subtree, results);
 		}
 
-		// eslint-disable-next-line guard-for-in, no-restricted-syntax
-		for (const blob in tree.blobs) {
-			results[`${base}${blob}`] = tree.blobs[blob];
+		for (const [blob, snapshot] of Object.entries(tree.blobs)) {
+			results[`${base}${blob}`] = snapshot;
```

Comment:
Suggestion for clarifying naming:

```suggestion
		for (const [blobName, blobId] of Object.entries(tree.blobs)) {
			results[`${base}${blobName}`] = blobId;
```